src/status_im/utils/number.cljs

```diff
@@ -13,3 +15,37 @@
   (let [scale (Math/pow 10 decimal-places)]
     (/ (Math/round (* n scale))
        scale)))
+
+(defn- caught-invalid-number []
+  (throw (js/Error "Invalid Number")))
+
+(defn- number-to-nk [number-string capitalize?]
+  (let [parsed-number (js/parseInt number-string)
```

Comment:
It's generally recommended to add the radix when parsing numbers. In this example, it'd be `(js/parseInt number-string 10)`. There are certain strings that are parsed differently in different runtimes.

Reply:
Okay